diff --git a/Angular/Abijith ps/Chapter 2/Activity/Activity b/Angular/Abijith ps/Chapter 2/Activity/Activity
--- a/Angular/Abijith ps/Chapter 2/Activity/Activity	
+++ b/Angular/Abijith ps/Chapter 2/Activity/Activity	
@@ -1 +1 @@
-Subproject commit e19d493f1c1eab788f223330178e989beb4ea4a7
+Subproject commit e19d493f1c1eab788f223330178e989beb4ea4a7-dirty
diff --git a/Angular/Abijith ps/Chapter 2/Exercise2/Exercise2 b/Angular/Abijith ps/Chapter 2/Exercise2/Exercise2
--- a/Angular/Abijith ps/Chapter 2/Exercise2/Exercise2	
+++ b/Angular/Abijith ps/Chapter 2/Exercise2/Exercise2	
@@ -1 +1 @@
-Subproject commit 65715375e5c1047689f66a2a078ad7331d15dfc0
+Subproject commit 65715375e5c1047689f66a2a078ad7331d15dfc0-dirty
diff --git a/Angular/Abijith ps/Chapter 2/workshop2/interviewShortList b/Angular/Abijith ps/Chapter 2/workshop2/interviewShortList
--- a/Angular/Abijith ps/Chapter 2/workshop2/interviewShortList	
+++ b/Angular/Abijith ps/Chapter 2/workshop2/interviewShortList	
@@ -1 +1 @@
-Subproject commit a8209b8378fefa5252b2457df3a6c5f1f5f2539d
+Subproject commit a8209b8378fefa5252b2457df3a6c5f1f5f2539d-dirty
